.phpcs.xml.dist

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0"?>
+<ruleset
+		name="VIP Go Sites"
+		xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		xsi:noNamespaceSchemaLocation="vendor/squizlabs/php_codesniffer/phpcs.xsd">
+	<description>Custom ruleset for VIP Go sites.</description>
+
+	<!-- Note: this file is only used for local PHPCS checks; the vip-go-ci bot does not read it. -->
+
+	<!-- For help in understanding this file: https://github.com/squizlabs/PHP_CodeSniffer/wiki/Annotated-ruleset.xml -->
+	<!-- For help in using PHPCS: https://github.com/squizlabs/PHP_CodeSniffer/wiki/Usage -->
+	<!-- For VIP GO code review items: https://vip.wordpress.com/documentation/vip-go/code-review-blockers-warnings-notices/ -->
+	<!-- For VIP GO PHPCS severity levels: https://vip.wordpress.com/documentation/phpcs-review-feedback/ -->
+
+	<!-- What to scan -->
+	<file>./themes/</file>
+	<!-- Ignoring Files and Folders:
+		https://github.com/squizlabs/PHP_CodeSniffer/wiki/Advanced-Usage#ignoring-files-and-folders -->
+	<exclude-pattern>./themes/twentyseventeen/</exclude-pattern>
```

Comment:
Let's refresh this theme path to `./themes/twentytwentyone/` as that's the current default theme included in this repo.